superset/databases/api.py

```diff
@@ -1039,3 +1040,40 @@ def validate_parameters(self) -> FlaskResponse:
         command = ValidateDatabaseParametersCommand(g.user, payload)
         command.run()
         return self.response(200, message="OK")
+
+    @expose("/upload_enabled/", methods=["GET"])
+    @protect()
+    @statsd_metrics
+    @event_logger.log_this_with_context(
+        action=lambda self, *args, **kwargs: f"{self.__class__.__name__}"
+        f".upload_enabled",
+        log_to_statsd=False,
+    )
+    def upload_enabled(self) -> Response:
+        """Returns a boolean saying whether uploads are enabled for the user
+        ---
+        get:
+          description:
+            Get the status of if any databases have allow file upload enabled
+            and also have a schema in a database that can have files uploaded to it.
+          responses:
+            200:
+              description: Database names
+              content:
+                application/json:
+                  schema:
+                    type: boolean
+            400:
+              $ref: '#/components/responses/400'
+            500:
+              $ref: '#/components/responses/500'
+        """
+        dbs_with_uploads = (
+            db.session.query(Database).filter_by(allow_file_upload=True).all()
```

Comment:
What happens if the ORM query fails? Can we add try catch to manage what we'd return to the user?

Reply:
You could prolly just wrap it in a try/catch with a `SupersetException` then return a SIP-40 compliant error

Reply:
What would a SIP-40 compliant error be? Also it is currently failing a test with:
 No fallback response defined for GET to http://localhost/api/v1/database/upload_enabled".]

seems to be in a fetchMock. Are those related?